package.json

```diff
@@ -40,14 +40,10 @@
     "@rollup/plugin-node-resolve": "^13.2.1",
     "@rollup/plugin-replace": "^4.0.0",
     "@sheerun/mutationobserver-shim": "^0.3.3",
-    "@storybook/addon-a11y": "^7.6.17",
-    "@storybook/addon-actions": "^7.6.17",
-    "@storybook/addon-controls": "^7.6.17",
-    "@storybook/addon-essentials": "^7.6.17",
-    "@storybook/addon-links": "^7.6.17",
-    "@storybook/addon-viewport": "^7.6.17",
-    "@storybook/react": "^7.6.17",
```

Comment:
`"@storybook/react"` is a dependency of `"@storybook/react-vite"` so having a direct dependency is redundant.